### PR TITLE
Copy environment toggle targeting

### PR DESCRIPTION
**Pull request by arthursoas**
To allow copy all feature toggle targeting between two environments, a script was developed to turn the activity easier. 
It will copy all feature toggles targeting and log detailed execution logs on a text file.

### CODE REVIEW

File: scripts/copy-environment-targeting/src/logger.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+export class Logger {
+    private filePath = "./src/output/logs.txt";
+    private file: number;
+
+    constructor () {
+        this.file = fs.openSync(this.filePath, "a", );
+    }
+
+    async logInfoAsync(message: string) {
```
Comment (vinioo) [inline scripts/copy-environment-targeting/src/logger.ts:11]: maybe you could have a function that receives the log level as a parameter. But i don't know if its necessary as this script is simple

File: scripts/copy-environment-targeting/package.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    "start": "ts-node ./src/index.ts",
+    "build": "tsc"
+  },
+  "author": "Arthur Soares",
+  "license": "MIT",
+  "dependencies": {
+    "axios": "^0.21.1",
+    "launchdarkly-api-typescript": "5.1.0",
+    "ts-node": "^10.0.0",
+    "typescript": "4.3.2"
```
Comment (vinioo) [inline scripts/copy-environment-targeting/package.json:16]: move typescript and ts-node to dev dependencies

File: scripts/copy-environment-targeting/src/featureToggleManager.ts [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+export class FeatureToggleManager {
+    private launchDarklyClient: LaunchDarklyClient;
+    private logger: Logger;
+    private project: string;
+    private sourceEnvironment: string;
+    private targetEnvironment: string;
+
+    constructor(
+        launchDarklyClient: LaunchDarklyClient,
```
Comment (vinioo) [inline scripts/copy-environment-targeting/src/featureToggleManager.ts:14]: you can use the [constructor shorthand](https://dev.to/satansdeer/typescript-constructor-shorthand-3ibd) here if you want

**Review by vinioo — APPROVED** [review]

File: scripts/copy-environment-targeting/src/featureToggleManager.ts (hunk 3 above)
Comment (arthursoas) [reply scripts/copy-environment-targeting/src/featureToggleManager.ts]: nice, didnt know about that

**Review by nicolasamaduro — APPROVED** [review]

**Pull request merged**